Yoast/wordpress-seo
# How to block time duration improvements

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Time duration needed is now a human readable string.

## Test instructions

This PR can be tested by following these steps:

* Insert a time duration in the how to block.
* Publish
* Verify that in the frontend (the post) it looks like a human readably string (i.e. 3 hours 1 minute = three hours and a minute).
* **NOTE** that human readable string will round some values, which may not be what we want. For now, won't fix because this PR needs to be tested and implemented in the RC.

The main way to see the difference is the human readable string for the duration.

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/10652
